microsoft-edge/devtools-guide-chromium/whats-new/2022/09/devtools-106.md

```diff
@@ -0,0 +1,179 @@
+
+To understand how DevTools fetches and caches your sourcemaps, watch the **Load status** column in the [Source Maps Monitor tool](../../../source-maps-monitor/source-maps-monitor-tool.md).  When you first load the webpage, the **Source Maps Monitor** tool shows that the source maps aren't cached yet:
+
+![Source Maps Monitor tool, upon first load](./devtools-106-images/source-maps-indexeddb-first-load.png)
+
+When you refresh the webpage, the **Source Maps Monitor** tool shows that the cached source maps are used:
+
+![Source Maps Monitor tool, upon second load](./devtools-106-images/source-maps-indexeddb-second-load.png)
+
+To clear the IndexedDB cache and force DevTools to re-load sourcemaps, run the **Clear source maps cache** command in the Command Menu.  See [Run commands in the Command Menu](../../../command-menu/index.md).
+
+For history, see [Issue 89](https://github.com/MicrosoftEdge/DevTools/issues/89).
+
+See also:
+* [Source maps are now cached with IndexedDB](../../2022/04/devtools-101.md#source-maps-are-now-cached-with-indexeddb) in _What's New in DevTools (Microsoft Edge 101)_
+
+
+<!-- ====================================================================== -->
+## Chromium color themes have been deprecated
+
+<!-- Subtitle: Users of Chromium themes in DevTools will be automatically migrated to the default dark or light theme for a more reliable experience. -->
+
+To improve reliability, the Chromium Light and Chromium Dark themes have been deprecated.  Users who previously had these themes selected will be automatically migrated to the **Light+** or **Dark+** theme, respectively:
+
+DevTools in **Light+** theme: 
+
+![DevTools in Light+ theme](./devtools-106-images/light-plus-theme.png)
+
+DevTools in **Dark+** theme:
+
+![DevTools in Dark+ theme](./devtools-106-images/dark-plus-theme.png)
+
+There are many DevTools color themes to choose from, such as **Monokai** and **Solarized**.  To change the color theme in DevTools, click the **Settings** (![Settings icon](../../../media/settings-gear-icon-light-theme.png)) button, and then in the **Preferences** page, in the **Theme** dropdown menu, select a theme:
+
+![Setting the DevTools theme to Monokai](./devtools-106-images/set-theme-monokai.png)
+
+See also:
+* [Apply a color theme to DevTools](../../../customize/theme.md)
+
+
+<!-- ====================================================================== -->
+## Accessibility improvements in Focus Mode and high contrast mode
+
+<!-- Subtitle: The new Dock location and Activity Bar location buttons in Focus Mode now work better with screen readers, and computed styles are easier to see in high contrast mode. -->
+
+
+#### Dock location and Activity Bar location buttons in Focus Mode now work better with screen readers
```

Comment:
Why are these `####` as opposed to `###`? Are you trying to more deeply nest these sections?

Reply:
No, not trying to deeper nest them.  Skipping h3's is not for logical / semantic reasons, it's deliberate & cosmetic, to effectively communicate that these are child sections by making them perceptibly smaller font than h2's.  In latest .css, h3s are almost as large as h2s.  This skipping of odd h levels is tested & approved/permitted.  Doesn't cause a doc build warning.

Reply:
Closing as By Design.